### PR TITLE
add placementrule to jobs

### PR DESCRIPTION
**Pull request by kuanf**
**What this PR does / why we need it**:
To add multicloud-operators-placementrule into the prow job

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

NA

**Specify your PR reviewers**:

/cc @gyliu513 @morvencao

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: prow/cluster/jobs/IBM/multicloud-operators-placementrule/OWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+approvers:
+- gyliu513
```
Comment (gyliu513) [inline prow/cluster/jobs/IBM/multicloud-operators-placementrule/OWNERS]: @kuanf you can set yourself as approver here and remove the three of us. ;-)

File: prow/cluster/jobs/IBM/multicloud-operators-placementrule/IBM.multicloud-operators-placementrule.master.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    decorate: true
+    always_run: true
+    path_alias: github.com/IBM/multicloud-operators-placementrule
+    rerun_command: /test build
+    spec:
+      containers:
+      - command:
+        - make
+        - build
+        image: quay.io/IBM/multicloud-operators-placementrule:2019-09-25T13-11-16
```
Comment (morvencao) [inline prow/cluster/jobs/IBM/multicloud-operators-placementrule/IBM.multicloud-operators-placementrule.master.yaml]: Can we reuse the current build-tools image?

Comment (gyliu513) [reply prow/cluster/jobs/IBM/multicloud-operators-placementrule/IBM.multicloud-operators-placementrule.master.yaml]: @kuanf

**Pull request merged**